tsconfig.json

```diff
@@ -9,16 +9,28 @@
     "baseUrl": ".",
     "paths": {
       "@polkadot/extension": ["packages/extension/src"],
+      "@polkadot/extension/esm": ["packages/extension/src"],
```

Comment:
Not really needed, but sanity-wise simpler if all the imports does the same thing.